doc/source/examples/dem/packing-in-circle/packing-in-circle.rst

```diff
@@ -94,26 +90,23 @@ The ``insertion info`` subsection manages the insertion of particles.
       set insertion prn seed                             = 19
     end
 
-First, the ``insertion method`` is selected. There are two insertion methods (``uniform`` and ``non_uniform``) in Lethe-DEM. In ``uniform`` insertion, the particles are inserted uniformly (without randomness in their initial location), while in ``non_uniform``, particles are inserted randomly in the insertion box. ``inserted number of particles at each time step`` specifies the desired number of particles to be inserted at each insertion step.
-
-.. note ::
-	The meaning of randomness in the initial location of particles in a non-uniform insertion is structured randomness (Using pseudo-random number generator algorithms).
+First, the ``insertion method`` is selected. In ``volume`` insertion, the particles are inserted in an insertion box. The ``inserted number of particles at each time step`` specifies the desired number of particles to be inserted at each insertion step.
 
 .. note::
     If the insertion box is not adequately large to insert ``inserted number of particles at each time step`` particles with the defined arrangement (initial distance between the inserted particles), Lethe prints a warning and inserts the maximum number of particles that fit inside the insertion box at each insertion step.
 
-``insertion frequency`` specifies the frequency of insertion steps. For example, if we set ``insertion frequency = 1000``, steps 0, 1000, 2000, 3000, ... will be defined as insertion iterations. Then we specify the dimensions of the insertion box. The box is defined using its ``minimum x``, ``minimum y``, ``maximum x``, and ``maximum y`` in two-dimensional simulations. In three-dimensional simulations, ``minimum z``, and ``maximum z`` are defined as well.
+``insertion frequency`` specifies the frequency of insertion steps. For example, if we set ``insertion frequency = 1000``, steps 0, 1000, 2000, 3000, ... will be defined as insertion iterations. 
 
 .. note::
-    We recommend that the defined insertion box have at least a distance of :math:`{d^{max}_p}` (maximum diameter of particles) from the triangulation boundaries. Otherwise, particles may have an overlap with the triangulation walls in the insertion.
+    We recommend that the defined insertion box have at least a distance of :math:`{d^{max}_p}` (maximum diameter of particles) from the triangulation boundaries. Otherwise, particles may be inserted with anoverlap with the triangulation walls.
```

Comment:
```suggestion
    We recommend that the defined insertion box have at least a distance of :math:`{d^{max}_p}` (maximum diameter of particles) from the triangulation boundaries. Otherwise, particles may be inserted with an overlap with the triangulation walls.
```